ollama/_client.py

```diff
@@ -97,6 +97,38 @@ def _request_stream(
   ) -> Union[Mapping[str, Any], Iterator[Mapping[str, Any]]]:
     return self._stream(*args, **kwargs) if stream else self._request(*args, **kwargs).json()
 
+  @overload
+  def generate(
+    self,
+    model: str = '',
+    prompt: str = '',
+    system: str = '',
+    template: str = '',
+    context: Optional[Sequence[int]] = None,
+    stream: bool = False,
```

Comment:
This is incorrect since it creates an overlap between the two overloads

```suggestion
    stream: Literal[False] = False,
```